pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -53,7 +53,8 @@ import (
 )
 
 const (
-	controllerName = "ceph-fs-subvolumegroup-controller"
+	controllerName             = "ceph-fs-subvolumegroup-controller"
+	cephSVGFileSystemNameIndex = "subvolumeGroupName/FilesystemName"
```

Comment:
is it possible to change this to `FilesystemName/subvolumeGroupName`?

Reply:
Yes

Reply:
can we change it in that case?

Reply:
what do you suggest?

Reply:
done